repos/fdbt-site/src/pages/api/searchOperators.ts

```diff
@@ -100,39 +68,32 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                     id: searchInputId,
                 });
             } else {
+                updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, { selectedOperators });
                 redirectTo(res, `/searchOperators?searchOperator=${refinedSearch}`);
                 return;
             }
         }
 
-        if (continueButtonClick) {
-            if (operatorsToRemove) {
-                errors.push({
-                    errorMessage: "Click the 'Remove Operator(s)' button to remove operators",
-                    id: removeOperatorsErrorId,
-                });
-            }
-            if (operatorsToAdd) {
-                const previousSearch = req.headers.referer?.split('?searchOperator=')[1] || '';
-                errors.push({
-                    errorMessage: "Click the 'Add Operator(s)' button to add operators",
-                    id: addOperatorsErrorId,
-                });
-                updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, { selectedOperators, errors });
-                redirectTo(res, `/searchOperators?searchOperator=${previousSearch}`);
-                return;
-            }
-        }
-
         if (errors.length > 0) {
             updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, { selectedOperators, errors });
             redirectTo(res, '/searchOperators');
             return;
         }
 
-        updateSessionAttribute(req, MULTIPLE_OPERATOR_ATTRIBUTE, { selectedOperators });
-
         if (continueButtonClick) {
+            if (!userSelectedOperators) {
+                selectedOperators = [];
+                errors.push({
+                    errorMessage: `Select at least one operator`,
+                    id: removeOperatorsErrorId,
```

Comment:
I think this error id needs changing to `searchInputId` because when it happens, the only thing we can link the error to is the search bar.

Reply:
@SamerIssam needs addressing still